### DISCUSSION TITLE
ipv6路由如何配置

### DISCUSSION THREAD

**Discussion by wanghuibin0** [XTLS/Xray-core#494] [Q&A]
比如我想使ipv6全部直连，应该怎样写路由规则？我查找了xray的教程，没有发现这方面的信息……是还不支持吗？

**Comment by wanghuibin0** [comment]
help me, please!

**Reply by itspig** [reply]
可以参考 
1.2.1. 指定出站 IP 
https://toutyrater.github.io/app/netflix.html
